cosmicds/viewers/dotplot/layer_artist.py

```diff
@@ -53,10 +60,24 @@ def __init__(self, view, viewer_state, layer_state=None, layer=None):
         add_callback(self._viewer_state, 'hist_n_bin', self._update_size)
         add_callback(self._viewer_state, 'viewer_height', self._update_size)
         add_callback(self._viewer_state, 'viewer_width', self._update_size)
```

Comment:
This isn't a problem with this PR, but this currently doesn't quite work because `viewer_height` and `viewer_width` were removed from the dotplot viewer state when the support for viewers knowing their dimensions was added to `cds_viewer` (pretty sure this was my fault). It'll be fine with the Hubble dotplot viewer (which does get the `cds_viewer` wrapper), but we should add in `viewer_height` and `viewer_width` properties to the dotplot viewer state so that this works as a standalone thing.

Reply:
Alright, I've added those back here 907b00ae8dbb38430c363a7d89526af28f88b4b5